Sources/LiveKit/Participant/Participant.swift

```diff
@@ -53,7 +53,7 @@ public class Participant: NSObject, ObservableObject, Loggable {
     public var joinedAt: Date? { _state.joinedAt }
 
     @objc
-    public var tracks: [String: TrackPublication] { _state.tracks }
+    public var tracksPublications: [Sid: TrackPublication] { _state.tracks }
```

Comment:
nit: in the JS version this is called `trackPublications` with the plural `s` only on publications. I think it would be good to align this across platforms

Reply:
Oops, this is a refactoring typo